modules/ROOT/pages/sid-reference.adoc

```diff
@@ -89,34 +91,74 @@ This is an example of the log, which shows one violation was detected in the col
 }
 ----
 
-////
+
 == SID Summary Log
 
-When you apply the WAF policy to Runtime Fabric, a `rtfSidSummary` log is generated, which shows the offending API, rule description, direction, and count for the number of times the SID rule was triggered. 
-This is an example of the log:
+When you apply the WAF policy to Runtime Fabric and select the *Detect sensitive information* option, a `rtfSidSummary` log is generated. The summary log contains a list of rule instances (rule ID, API, method, direction, and count) where at least one trigger occurred during the one-minute collection interval. 
+
+The count applies only to messages (request or response) and is how many messages in the collection interval triggered the rule instance. 
+
+For example, if the request is for a birth certificate application, and only one request occurred in the last minute, and the request contained:
+
+----
+DOB of Mother
+SSN of Mother
+DOB of Father
+SSN of Father
+DOB of Father
+DOB of Baby
+CCN to pay for the application
+----
+
+The summary message would contain three items--one for each rule violated (DOB, SSN, and CCN). In other words, every rule violation in one message is counted only once, even if there are multiple violations of the same rule in one message.  
+
+This is an example of the `rtfSidSummary` log for a request where sensitive information (tax ID rule) was detected in the payload.
 
 [json]
 ----
 {
-	"timeStamp": "2019-04-15T16:13:00.045Z",
+	"timeStamp": "2019-05-03T20:28:00.493Z",
 	"collectionIntervalInMinutes": "1",
-	"node": "node-host-name",
-	"sidSummary": {
-		"api": "/Post",
-		"rule": "TAXID",
+	"node": "node-host",
+	"sidSummary": [{
+		"api": "myapi",
+		"envId": "a2f87a26-6ee9-4843-a145-bf258e630aa8",
+		"method": "POST",
+		"rule": "Social Security Number key value pair",
 		"direction": "Request",
 		"count": 1
-	}
+	}]
 }
 ----
- 
-////
+
+== Supported Content Types
+
+SID rules match the following content-types for sensitive data tags and values:
+
+* Request Types: 
+ ** text/plain
+ ** application/json
+ * Response Types:  
+  ** text/plain 
+  ** application/json
+
+== Rule Categories
+
+Sensitive information detection rules belong to rule categories, which include: 
+
+* PII: Personally identifiable information, such as date of birth, social security number, passport number, and so on. 
+* PCI: Payment card information, such as a Visa or Mastercard number. 
+* PHI: Protected health information.
 
 == View Rule Categories and IDs in RAML
```

Comment:
@valkyrie69, I checked the RAML file, there's no info for "SID categories". I think we should change the title to 
"View Rule Name and Rule Id" 
Here's an example of the content in the RAML file. 
`        
{
          "id": 600780,
          "ver": "",
          "rev": "1",
          "phase": "response",
          "severity": "WARNING",
          "accuracy": "5",
          "maturity": "1",
          "taglist": [
            "MULESOFT_SID"
          ],
          "msg": "VISA CARD",
          "variables": "RESPONSE_BODY",
          "operator": "UkVTUE9OU0VfQk9EWSAoPzwhWy5cdy1dKTRbMC05XXszfShbIC1dPylbMC05XXs0fVwxWzAtOV17NH1cMVswLTldezR9KD8hWy5cdy1dKQo=",
          "paranoia": 1,
          "chain": "no"
        },
`